src/containers/Header/PreviewFlag.js

```diff
@@ -1,8 +1,20 @@
 import React from 'react';
+import {useTranslation} from "react-i18next";
 
 export default function PreviewFlag({ name }) {
-    if(!name) {
-        return null
+    const flagName = name?.toLowerCase();
+    const {t} = useTranslation();
```

Comment:
@queuing4oranges Add spaces to the curly brackets - `const { t } = useTranslation();`